data/json/materials.json

```diff
@@ -906,5 +882,25 @@
       { "fuel": 1, "smoke": 1, "burn": 2 },
       { "fuel": 1, "smoke": 1, "burn": 5 }
     ]
+  },
+  {
+    "type": "material",
+    "ident": "feces",
+    "name": "Feces",
+    "density": 4,
+    "bash_resist": 1,
+    "cut_resist": 1,
+    "acid_resist": 1,
+    "fire_resist": 0,
+    "elec_resist": 1,
+    "chip_resist": 1,
+    "dmg_adj": [ "squished", "squshed", "mashed", "mushed" ],
```

Comment:
Although "squshed" is technically a word, it's incredibly rare. Maybe "squashed" ?